env0/env0-backstage-plugin
# Feat: Publish npm packages

Signed-off-by: elbaz <eranelbaz97+github@gmail.com>

### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
We would like to publish our packages
### Solution
Use `npm publish` for each plugin we have

`build:all` requires first `tsc:all`
Then publish our packages

Added release drafter 
### QA

Followed [Publish a npm package locally for testing | by Debashish Pal | Medium](https://medium.com/@debshish.pal/publish-a-npm-package-locally-for-testing-9a00015eb9fd) to check if we publish correctly
